library/spec/pivotal-ui-react/toggle/toggle_spec.js

```diff
@@ -70,4 +70,35 @@ describe('Toggle', () => {
       expect('.toggle-switch').toBeChecked();
     });
   });
+
+  describe('size attribute', function() {
+    beforeEach(function() {
+      ReactDOM.unmountComponentAtNode(root);
+    });
+
+    it('renders with size=medium by default', function() {
+      ReactDOM.render(<Toggle id='boggle' />, root);
+      expect($('label').attr('class')).toContain('medium');
```

Comment:
I understand `subject` doesn't work for everyone, you don't need to use it. 

Looking at the original file, there are already some kinda weird testing patterns in there (namely some extra `unmountComponentAtNode`), but you didn't add, so I'll deal with those separately.